lib/components/src/form/input/input.tsx

```diff
@@ -92,27 +92,35 @@ const validation = ({ valid, theme }) => {
   }
 };
 
-export const Input = styled.input(styles, sizes, alignment, validation, { minHeight: 32 });
-Input.styles = { ...styleResets, ...styles };
-Input.sizes = sizes;
-Input.alignment = alignment;
+export const Input = styled.input<any>(styles as any, sizes, alignment as any, validation, {
+  minHeight: 32,
+});
+(Input as any).styles = { ...styleResets, ...styles };
+(Input as any).sizes = sizes;
+(Input as any).alignment = alignment;
```

Comment:
Is there anything we can do about all these `any` for `styled`?

Reply:
I tried to but have no solution for now... 😞